pkg/generators/models/go_type_from_ref.go

```diff
@@ -5,16 +5,17 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/rs/zerolog/log"
-
 	"github.com/getkin/kin-openapi/openapi3"
 
 	tpl "github.com/contiamo/openapi-generator-go/pkg/generators/templates"
 )
 
 func goTypeFromSpec(schemaRef *openapi3.SchemaRef) string {
 	if schemaRef == nil {
-		log.Fatal().Msg("got nil schema ref")
```

Comment:
No need to panic and exit the process here, I just initialize it now like we do in the rest of the code.